ts/parts/Pointer.ts

```diff
@@ -854,6 +843,39 @@ Highcharts.Pointer.prototype = {
         });
     },
 
+    /**
+     * Set inactive state to all series that are not currently hovered,
+     * or, if `inactiveOtherPoints` is set to true, set inactive state to
+     * all points within that series.
+     *
+     * @function Highcharts.Pointer#setInactiveSeries
+     *
+     * @private
+     *
+     * @param {Array<Highcharts.Point>} points
+     *        Currently hovered points
+     *
+     */
+    setInactiveSeries: function (
+        this: Highcharts.Pointer,
+        points: Array<Highcharts.Point>
```

Comment:
As indicated in the corresponding description, this should also support Series beside Point. It is a bit strange to have a function `setInactiveSeries` that takes points only.

Reply:
There's no need for `Series` array. From currently hovered points, retrieve all inactive series and set them inactive. You are right, we could use a better name - what do you suggest? `setInactiveSeriesFromPoints` is just too long. Description of the function is:
>     * Set inactive state to all series that are not currently hovered,
>     * or, if `inactiveOtherPoints` is set to true, set inactive state to
>     * all points within that series.

Reply:
How about `applyInactiveState`?

Reply:
Doesn't `applyInactiveState` miss `Series` keyword then? Maybe something like hmm.. `inactivateSeries`? I don't know.. what do you think @jon-a-nygaard , @oysteinmoseng , @sebastianbochan ?

Reply:
I do not think `Series` should be included in the name since it may also set the state on points. I think I prefer @bre1470 suggestion of `applyInactiveState` or plural `applyInactiveStates`.

Alternatively `setInactiveStates`, but I do not like that much since it is not setting the state on the points we pass in, and also it does not seem much like a regular setter function.

Reply:
You are right @bre1470 and @jon-a-nygaard - my mistake, where were my eyes.. 👀 Fix coming soon!